GUI/Main.cs

```diff
@@ -1031,11 +1035,13 @@ private void exportModListToolStripMenuItem_Click(object sender, EventArgs e)
 
         private void manageKspInstancesMenuItem_Click(object sender, EventArgs e)
         {
-            Instance.Manager.ClearAutoStart();
             var old_instance = Instance.CurrentInstance;
             var result = new ManageKspInstances(!actuallyVisible).ShowDialog();
             if (result == DialogResult.OK && !Equals(old_instance, Instance.CurrentInstance))
-                Instance.CurrentInstanceUpdated(false);
+            {
+                ModList.ClearSelection();
+                Instance.CurrentInstanceUpdated(true);
```

Comment:
(Old gripe) We don't need `Instance` here. This code is already running in the instance.